backends/text-to-speech/windows/sphelper-scummvm.h

```diff
@@ -0,0 +1,2648 @@
+/*******************************************************************************
+* SPHelper.h *
+*------------*
+*   Description:
+*       This is the header file for core helper functions implementation.
```

Comment:
Does this file have any changes from the original? If yes, which ones?

Reply:
It has changes, I had to make them, so it works with MinGW. I added a summary of the changes to the top of the file. I think documenting each changed line would be too much and it would end up being quite confusing when reading it.

Reply:
Thanks

Reply:
It is added by this commit: [e21ebee](https://github.com/scummvm/scummvm/pull/1808/commits/e21ebeeee7d8cb908d744febc97c6263f48c312d)